packages/subscriber/src/components/add-form/index.tsx

```diff
@@ -133,103 +140,107 @@ export const AddSubscriberForm: FunctionComponent< Props > = ( props ) => {
-											__(
-												'<strong>%s</strong> <uploadBtn>Replace</uploadBtn> | <removeBtn>Remove</removeBtn>'
-											),
-											selectedFile?.name
-										),
-										{
-											strong: createElement( 'strong' ),
-											uploadBtn: formFileUploadElement,
-											removeBtn: createElement( Button, {
-												isLink: true,
-												onClick: () => setSelectedFile( undefined ),
-											} ),
-										}
-									) }
-								</label>
-							) }
-
-							{ isSelectedFileValid && ! selectedFile && (
-								<label>
-									{ createInterpolateElement(
-										__(
-											'Or bring your mailing list from other newsletter services by <uploadBtn>uploading a CSV file.</uploadBtn>'
-										),
-										{ uploadBtn: formFileUploadElement }
-									) }
-								</label>
+						</label>
+					) }
+
+					{ isSelectedFileValid && selectedFile && (
+						<label className={ 'add-subscriber__form-label-links' }>
+							{ createInterpolateElement(
+								sprintf(
+									/* translators: the first string variable shows a selected file name, Replace and Remove are links" */
+									__(
+										'<strong>%s</strong> <uploadBtn>Replace</uploadBtn> | <removeBtn>Remove</removeBtn>'
+									),
+									selectedFile?.name
+								),
+								{
+									strong: createElement( 'strong' ),
+									uploadBtn: formFileUploadElement,
+									removeBtn: createElement( Button, {
+										isLink: true,
+										onClick: () => setSelectedFile( undefined ),
+									} ),
+								}
 							) }
-
-							<NextButton type={ 'submit' } className={ 'add-subscriber__form-submit-btn' }>
-								{ submitBtnName || 'Add subscribers' }
-							</NextButton>
-							{ showSkipBtn && (
-								<SkipButton
-									className={ 'add-subscriber__form-skip-btn' }
-									onClick={ () => onSkipBtnClick?.() }
-								>
-									Not yet
-								</SkipButton>
+						</label>
+					) }
+
+					{ showCsvUpload && isSelectedFileValid && ! selectedFile && (
+						<label>
+							{ createInterpolateElement(
+								__(
+									'Or bring your mailing list from other newsletter services by <uploadBtn>uploading a CSV file.</uploadBtn>'
+								),
+								{ uploadBtn: formFileUploadElement }
 							) }
-							<p className={ 'add-subscriber__form--disclaimer' }>
-								By adding a mailing list CSV, you are confirming that you have the rights to share
-								newsletters with the people within your list.{ ' ' }
-								<Button isLink={ true }>Learn more</Button>
-							</p>
-						</form>
-					</div>
-				</>
-			) }
+						</label>
+					) }
+
+					{ showCsvUpload && isSelectedFileValid && selectedFile && (
+						<p className={ 'add-subscriber__form--disclaimer' }>
+							By clicking "continue", you represent that you've obtained the appropriate consent to
+							email each person on your list. <Button isLink={ true }>Learn more</Button>
```

Comment:
I know this is in draft/in progress and you might've been already on it, but I noticed the PR misses quite a few `__()`.

Reply:
My idea was to leave that task as the last until we get the final UI design.
[The open task](https://github.com/Automattic/wp-calypso/issues/65540) for that exists, so we will not miss it.

Reply:
Would be great to get those strings locked today, so that we can send strings for translation on Monday and then we'd have one week turnaround time to get them back.

@saygunnyc do we have final strings for importers?

Reply:
Also, @saygunnyc, what should be the reference for the **Learn more** link?

Reply:
✔️ The string translations have been supported: https://github.com/Automattic/wp-calypso/pull/66998/commits/1b84802eeade4639781e750f28d5ba701023e667